bundles/org.palladiosimulator.simexp.dsl.smodel.interpreter/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/ModelsLookup.java

```diff
@@ -0,0 +1,35 @@
+package org.palladiosimulator.simexp.dsl.smodel.interpreter;
+
+import org.palladiosimulator.edp2.models.measuringpoint.MeasuringPoint;
+import org.palladiosimulator.experimentautomation.experiments.Experiment;
+import org.palladiosimulator.simexp.dsl.smodel.smodel.Probe;
+
+public class ModelsLookup implements IModelsLookup {
+    private final Experiment experiment;
+
+    public ModelsLookup(Experiment experiment) {
+        this.experiment = experiment;
+    }
+
+    @Override
+    public MeasuringPoint findMeasuringPoint(Probe probe) {
+        IMeasuringPointFinder finder = getFinder(probe);
+
+        MeasuringPoint find = finder.find(experiment, probe);
+        if (find == null) {
+            throw new RuntimeException(String.format("No monitor found for probe '%s' with id: '%s' found",
```

Comment:
Better return 
"No monitor found for probe '%s' with %s: '%s' found", probe.getName(), probe.getKind(), probe.getIdentifier()));